api/v1/multiclusterhub_methods.go

```diff
@@ -69,7 +70,8 @@ var MCHComponents = []string{
 	MCH,                // Adding MCH component to ensure legacy resources are cleaned up properly.
 	MultiClusterObservability,
 	Search,
-	SiteConfigPreview,
+	DeprecatedSiteConfigPreview, // Do we need this in the components list?
```

Comment:
Is this what we decided to get rid of in scrum yesterday?

Reply:
I believe so, yes. I want to bring it up in Scrum on Monday to make sure though

Reply:
We do not need to add a `DeprecatedSiteConfigPreview` variable, since `SiteConfig` was onboarded in this release. We just need to remove the `-preview` from the original `SiteConfig` variable.

Reply:
oh nice haha, even easier than I had thought